### ISSUE TITLE
Need help understanding the code

### ISSUE THREAD

**Issue by CarlosUvaSilva** [nhcamp/EPL-Betting#3]
Hello again @nhcamp 

I'm trying to replicate your results after you helped me getting the right files but I'm having a hard time doing so.

1. I don't understand why you are "skipping" a year in this code
```
train_df = double_df['1994-95':'2014-15']
test_df = double_df.loc['2016-17']
```

2. On your `did_I_make_money` method why are you defining Bournemouth as zero?
```
for index,stats in team_bets.iterrows():
            if (stats['HomeTeam'] == 'Bournemouth') | (stats['AwayTeam'] == 'Bournemouth'):
                profit_loss.append(0)
            elif stats['Model Prediction'] == 'Loss/Draw':
                profit_loss.append(0)
            elif (stats['Model Prediction'] == 'Win'):
                if (stats['HomeTeam'] == team) & (stats['FTR'] == 'H'):
                    profit_loss.append(stats['B365H'] * 100 - 100)
                elif (stats['AwayTeam'] == team) & (stats['FTR'] == 'A'):
                    profit_loss.append(stats['B365A'] * 100 - 100)
                else:
                    profit_loss.append(-100)
``` 

Also I've tried to select only (I'm not skipping a year bc I assumed that's a typo)
```
train_df = double_df['2000-2001':'2014-15']
test_df = double_df.loc['2016-17']
``` 

and I'm getting a 1900$ loss. Could you try to replicate these results?

Thank you very much

**Comment by nhcamp** [comment]
Hi Carlos. You've likely figured these out by now - I started a new job and returning to this repo slipped my mind as I got up to speed with things. But I figured I'd answer your questions so they sit here for anyone else wondering the same.

1. I didn't use the 15-16 season because I wanted to make sure I didn't have data leakage between my train and test set. I didn't know if the model's performance would be artificially inflated by including the league positions from a season in my training set but I didn't want to chance it. 

Additionally, the 15-16 season was topsy-turvy with Leicester winning the league at 5000-1 odds and several of the big teams under performing their historic averages. With such a small data-set, and the notion that we're predicting on repeating patterns and not the outrageous unpredictability that I love about soccer, it also makes sense to not use the 15-16 season in training a model.

2. Bournemouth's first ever season in the top flight was 2015-16. As we don't have any historic data for this team, it's wise to not make any bets on their performance because the model will predict they lose every game - meaning you lose a bunch of money every time they win.  

Hope you figured these out though and you've been raking in cash since!
